### PR TITLE
autopartitioning-eng

### PR DESCRIPTION
**Pull request by ElenaAfina**
### Changelog entry 

add autopartitioning

### Changelog category 

* Documentation (changelog entry is not required)

### Additional information

...

### CODE REVIEW

File: ydb/docs/en/core/yql/reference/yql-core/syntax/alter-topic.md [hunk 1]
```diff
@@ -141,3 +141,31 @@ ALTER TOPIC `my_topic` RESET (
...
+
+### Change autopartitioning modes for the topic {#autopartitioning}
+
+Next command set [autopartitioning](../../../../concepts/topic#autopartitioning) into the mode UP:
+
+```yql
+ALTER TOPIC `my_topic` SET (
+    min_active_partitions = 1,
+    max_active_partitions = 5,
+    auto_partitioning_strategy = 'scale_up'
```
Comment (blinkov) [inline ydb/docs/en/core/yql/reference/yql-core/syntax/alter-topic.md:153]: Two naming inconsistencies:

* above, we call these "modes", while the actual setting name is "strategy"
* above, we call this mode/strategy just "up", while the actual setting value is "scale up"

**Review by blinkov — DISMISSED** [review]

File: ydb/docs/en/core/yql/reference/yql-core/syntax/alter-topic.md (hunk 1 above)
Comment (ElenaAfina) [reply ydb/docs/en/core/yql/reference/yql-core/syntax/alter-topic.md:153]: 1. I'll change "mode" into the "strategy"? 
2. YQL will be fixed here, and after that both "scale up" & "up" will be possible, and we'll change the documentation after that (not in this PR).

File: ydb/docs/en/core/yql/reference/yql-core/syntax/alter-topic.md [hunk 2]
```diff
@@ -141,3 +141,31 @@ ALTER TOPIC `my_topic` RESET (
...
+
+The following command pauses the topic [autopartitioning](../../../../concepts/topic#autopartitioning):
+
+```yql
+ALTER TOPIC `my_topic` SET (
+    auto_partitioning_strategy = 'paused'
+);
+```
+
+The following command unpauses the topic [autopartitioning](../../../../concepts/topic#autopartitioning):
```
Comment (blinkov) [inline ydb/docs/en/core/yql/reference/yql-core/syntax/alter-topic.md:165]:
```suggestion
The following command sets the [autopartitioning](../../../concepts/topic.md#autopartitioning) strategy to `UP`:

```yql
ALTER TOPIC `my_topic` SET (
    min_active_partitions = 1,
    max_active_partitions = 5,
    auto_partitioning_strategy = 'scale_up'
);
```

The following command pauses the topic [autopartitioning](../../../concepts/topic.md#autopartitioning):

```yql
ALTER TOPIC `my_topic` SET (
    auto_partitioning_strategy = 'paused'
);
```

The following command unpauses the topic [autopartitioning](../../../concepts/topic.md#autopartitioning):
```

File: ydb/docs/en/core/yql/reference/yql-core/syntax/alter-topic.md [hunk 3]
```diff
@@ -141,3 +141,31 @@ ALTER TOPIC `my_topic` RESET (
 * `supported_codecs`: List of [codecs](../../../../concepts/topic#message-codec) supported by the topic. Value type: `String`.
 
 {% endif %}
+
+### Change autopartitioning strategies for the topic {#autopartitioning}
+
+The following command sets the [autopartitioning](../../../../concepts/topic#autopartitioning) strategy to `UP`:
```
Comment (blinkov) [inline ydb/docs/en/core/yql/reference/yql-core/syntax/alter-topic.md:147]:
```suggestion
The following command sets the [autopartitioning](../../../concepts/topic.md#autopartitioning) strategy to `UP`:
```

File: ydb/docs/en/core/yql/reference/yql-core/syntax/alter-topic.md [hunk 4]
```diff
@@ -141,3 +141,31 @@ ALTER TOPIC `my_topic` RESET (
...
+
+```yql
+ALTER TOPIC `my_topic` SET (
+    min_active_partitions = 1,
+    max_active_partitions = 5,
+    auto_partitioning_strategy = 'scale_up'
+);
+```
+
+The following command pauses the topic [autopartitioning](../../../../concepts/topic#autopartitioning):
```
Comment (blinkov) [inline ydb/docs/en/core/yql/reference/yql-core/syntax/alter-topic.md:157]:
```suggestion
The following command pauses the topic [autopartitioning](../../../concepts/topic.md#autopartitioning):
```

File: ydb/docs/en/core/yql/reference/yql-core/syntax/alter-topic.md (hunk 2 above)
Comment (blinkov) [inline ydb/docs/en/core/yql/reference/yql-core/syntax/alter-topic.md:165]:
```suggestion
The following command unpauses the topic [autopartitioning](../../../concepts/topic.md#autopartitioning):
```

**Review by fomichev3000 — APPROVED** [review]

**Pull request merged**